v2/emailpassword/common-customizations/sessions/with-jwt/jwt-verification.mdx

```diff
@@ -256,4 +256,22 @@ Claims like email verification and user roles claims are added to the access tok
 On claim validation failure, you must send a `403` to the frontend which will cause our frontend SDK (pre built UI SDK) to recheck the claims added on the frontend and navigate to the right screen.
 :::
 
+## Check for anti-csrf during authorization
+
+:::important
+You will need to check for anti-csrf for **NON** GET requests when cookie based authentication is enabled.
+::: 
+
+There are two methods for configuring [CSRF protection](../anti-csrf):
+- `VIA_TOKEN`
+- `VIA_CUSTOM_HEADER`
+
+### Checking for anti-csrf when `VIA_TOKEN` is set
+When configured with `VIA_TOKEN`, an explicit `anti-csrf` token will be attached to requests. To verify the `anti-csrf` token you will need to compare it the to value of the `antiCsrfToken` key from the payload of the decoded JWT.
+
+### Checking for anti-csrf when `VIA_CUSTOM_HEADER` is set
+When configured with `VIA_CUSTOM_HEADER`, you will need to check for the presence of the `rid` header from incoming requests. 
+
```

Comment:
how will the user know what is configured? They don't explicitly set this setting.. our sdk decides it for them.